### DISCUSSION TITLE
How to show help for a subcommand without requiring the parent command's arguments

### DISCUSSION THREAD

**Discussion by MarkKoz** [pallets/click#2178] [Q&A] [answered]
I have a usage like this:

```
program ARG subcommand [OPTIONS]
```

The only way I can see the help for `subcommand` is like so

```
program "some value for ARG" subcommand --help
```

This is not ideal because the value doesn't actually need to be read to display the help. It's only being passed to make click's argument parser understand that I want help for the subcommand. Is there a way to omit the requirement for arguments when `--help` is used?

**Comment by davidism** [comment, marked as answer]
https://github.com/pallets/click/issues/295#issuecomment-708129734
